docs/CHANGELOG.asciidoc

```diff
@@ -21,6 +22,53 @@ Review important information about the {kib} 8.x releases.
 
 --
 
+[[release-notes-8.1.1]]
+== {kib} 8.1.1
+
+coming::[8.1.1]
+
+Review the following information about the {kib} 8.1.1 release.
+
+[float]
+[[breaking-changes-8.1.1]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 8.1.1, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+
+There are no breaking changes in {kib} 8.1.1.
+
+// end::notable-breaking-changes[]
+
+To review the breaking changes in previous versions, refer to the following: 
+
+{kibana-ref-all}/8.0/release-notes-8.0.0.html#breaking-changes-8.0.0[8.0.0] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc2.html#breaking-changes-8.0.0-rc2[8.0.0-rc2] | {kibana-ref-all}/8.0/release-notes-8.0.0-rc1.html#breaking-changes-8.0.0-rc1[8.0.0-rc1] | {kibana-ref-all}/8.0/release-notes-8.0.0-beta1.html#breaking-changes-8.0.0-beta1[8.0.0-beta1] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha2.html#breaking-changes-8.0.0-alpha2[8.0.0-alpha2] | {kibana-ref-all}/8.0/release-notes-8.0.0-alpha1.html#breaking-changes-8.0.0-alpha1[8.0.0-alpha1]
+
+[float]
+[[enhancement-v8.1.1]]
+=== Enhancement
+Dashboard::
+Improves controls empty state {kibana-pull}125728[#125728]
+
+[float]
+[[fixes-v8.1.1]]
+=== Bug fixes
+Data ingest::
+The dot expander processor in the Ingest Pipelines UI now allows setting a wildcard (`*`) for the field parameter {kibana-pull}123522[#123522]
+
+Elastic Security::
+For the Elastic Security 8.1.1 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Enterprise Search::
+For the Elastic Enterprise Search 8.1.1 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+
+Fleet::
+* Adds a new validation message {kibana-pull}127239[#127239]
+* Fixes empty assets on package install {kibana-pull}127070[#127070]
+* Hide enroll command when clicking on create agent policy {kibana-pull}126431[#126431]
```

Comment:
The gerund form here is a little odd because "clicking" makes it sound it's happening more than once. I usually shy away from saying "user" but it might help here. How about something like:

Hide enroll command when user creates a new agent policy in the Add agent flyout.